libs/ui/layout/src/lib/anchor-link/anchor-link.directive.ts

```diff
@@ -11,20 +12,29 @@ import {
 @Directive({
   selector: '[gnUiAnchorLink]',
 })
-export class AnchorLinkDirective implements OnInit, OnDestroy {
+export class AnchorLinkDirective
+  implements OnInit, AfterViewChecked, OnDestroy
+{
   @Input('gnUiAnchorLink') targetId: string
   @Input('gnUiAnchorLinkDisabledClass') disabledClass: string
   @Input('gnUiAnchorLinkEnabledClass') enabledClass: string
+  @Input('gnUiAnchorLinkInViewClass') inViewClass: string
```

Comment:
Remember to include this new input in the associated storybook entry :)

Reply:
Good point, I've added it.